app/models/user.rb

```diff
@@ -101,8 +101,10 @@ def full_name
   end
 
 
-  def grant_membership
+  # Need to be able to turn off 'sending email' during seeding
+  def grant_membership(send_email: true)
     update(member: true, membership_number: issue_membership_number)
+    MemberMailer.membership_granted(self).deliver if send_email
```

Comment:
On second review I'd like to hold off on this PR for now - until we work out how and where we determine we will grant membership.  

Currently, this method is being called from the webhook that handles HIPS payments updates.  That webhook should operate as fast as possible - in order to complete required processing by the time HIPS sends control back to us - and sending an email might jeopardize that.

Reply:
Good point. Let's talk about this in Slack. :-)

Reply:
As discussed, I agree with going ahead with this PR.